components/locid/src/extensions/mod.rs

```diff
@@ -73,7 +73,7 @@ pub enum ExtensionType {
 }
```

Comment:
Optional: find and replace areas in our current docs strings where we can switch from runtime parsing to compile-time parsing. For example, in components/calendar/src/any_calendar.rs:

```rust
/// let locale = Locale::from_str("en-u-ca-japanese").unwrap();
```

can become

```rust
/// let locale = locale!("en-u-ca-japanese"); 
```

I also see examples in

- components/datetime/src/any/date.rs
- components/datetime/src/any/datetime.rs
- components/datetime/src/any/zoned_datetime.rs
- components/locid/src/locale.rs
- provider/adapters/src/fallback/mod.rs

Reply:
Maybe do this is a follow up

Reply:
Will send a separate PR to add these.

Reply:
OK; make sure this is tracked somewhere / not forgotten

Reply:
Addressing these in #2401